lib/sql_footprint/sql_anonymizer.rb

```diff
@@ -3,6 +3,7 @@ class SqlAnonymizer
     GSUBS = {
       /\sIN\s\((.*)\)/ => ' IN (values-redacted)'.freeze, # IN clauses
       /\s\=\s([0-9]+)/ => ' = number-redacted'.freeze, # numbers
+      /\s\!\=\s([0-9]+)/ => ' != number-redacted'.freeze, # numbers w/ not equals
```

Comment:
We would also need to rewrite some of the code for the sql anonymizer, but I don't think that would be difficult.